src/transformers/models/llama/modeling_flax_llama.py

```diff
@@ -434,7 +434,7 @@ def init_weights(self, rng: jax.random.PRNGKey, input_shape: Tuple, params: Froz
             self._missing_keys = set()
             return freeze(unflatten_dict(params))
         else:
-            return random_params
+            return freeze(random_params)
```

Comment:
Let's leave this as is and update the test in a follow-up PR
```suggestion
            return random_params
```

Reply:
I've removed it, but when it's not available the dtype conversion tests fail `FlaxMistralModelTest::test_to_bf16 ` and `FlaxMistralModelTest::test_to_fp16`, while if they're returned as frozen they don't fail